docs/addons/karpenter.md

```diff
@@ -53,12 +49,11 @@ const karpenterAddonProps = {
   weight: 20,
   interruptionHandling: true,
 }
-const vpcCniAddOn = new blueprints.addons.VpcCniAddOn();
+
 const karpenterAddOn = new blueprints.addons.KarpenterAddOn(karpenterAddonProps);
-const addOns: Array<blueprints.ClusterAddOn> = [ vpcCniAddOn, karpenterAddOn ];
 
 const blueprint = blueprints.EksBlueprint.builder()
-  .addOns(...addOns)
+  .addOns(...karpenterAddOn)
```

Comment:
remove ..., it is not needed for a single addon. varargs functions like addOns() allow the following:
```
.addOns(firstAddOn)
.addOns(secondAddOn)
.addOns(firstAddOn, secondAddOn)
.addOns(...arrayOfAddOns)
```

Reply:
addressed.